config/overrides.json

```
@@ -147,5 +147,8 @@
   },
   "/Lotus/Types/Recipes/Kubrow/Collars/PrimeKubrowCollarABuckleComponent": {
     "ducats": 65
+  },
+  "/Lotus/Weapons/Tenno/Pistol/LatoVandal": {
+    "tradable": false
   }
```

Could you also add an override for braton vandal?
```suggestion
  },
  "/Lotus/Weapons/Tenno/Rifle/VIPRifle": {
    "tradable": false
  }
```

---

sure